tests/platform_helper/utils/test_validation.py

```diff
@@ -940,58 +940,66 @@ def test_validate_database_copy_multi_postgres_failures(capfd):
     )
 
 
+@pytest.mark.parametrize(
+    "config, expected_response",
+    [
+        (
+            # No engine defined in either env
+            {
+                "extensions": {
+                    "connors-redis": {
+                        "type": "redis",
+                        "environments": {"*": {"plan": "tiny"}, "prod": {"plan": "largish"}},
+                    }
+                },
+            },
+            "",
+        ),
+        (
+            # Valid engine version defined in *
+            {
+                "extensions": {
+                    "connors-redis": {
+                        "type": "redis",
+                        "environments": {
+                            "*": {"engine": "7.1", "plan": "tiny"},
+                            "prod": {"plan": "tiny"},
+                        },
+                    }
+                },
+            },
+            "",
+        ),
+        (
+            # Invalid engine defined in prod environment
+            {
+                "extensions": {
+                    "connors-redis": {
+                        "type": "redis",
+                        "environments": {
+                            "*": {"plan": "tiny"},
+                            "prod": {"engine": "6.1", "plan": "tiny"},
```

Comment:
6.1 looks like it could be OK, this could make it a bit clearer...
```suggestion
                            "prod": {"engine": "invalid", "plan": "tiny"},
```